Packs/AbnormalSecurity/Integrations/AbnormalSecurityEventCollector/AbnormalSecurityEventCollector.py

```diff
@@ -0,0 +1,103 @@
+from CommonServerPython import *
+import demistomock as demisto
+
+VENDOR = 'Abnormal Security'
+PRODUCT = 'Email Protection'
+
+
+class Client(BaseClient):
+    def get_threats(self, params):
+        return self._http_request('GET', params=params, url_suffix='threats')
+
+    def get_threat(self, threat):
+        return self._http_request('GET', url_suffix=f'threats/{threat}')
+
+
+def get_threats(client: Client, after):
+    before = get_timestamp_format('now')
+    threats_ids = get_list_threats(client, after, before)
+    messages = []
+    if threats_ids:
+        for threat in reversed(threats_ids):
+            messages += get_messages_by_datetime(client, threat.get('threatId'), after, before)
+        ordered_messages = sorted(messages, key=lambda d: d['receivedTime'])
+        return ordered_messages, before
+    return [], before
+
+
+def get_messages_by_datetime(client: Client, threat_id, after, before):
+    messages = []
+    res = client.get_threat(threat_id)
+    for message in res.get('messages'):
```

Comment:
Are the messages ordered from newest to oldest?

Reply:
yes